docs/examples/eventrouter.yaml

```diff
@@ -26,69 +26,43 @@ spec:
           serviceAccountName: addon-manager-workflow-installer-sa
           templates:
           - name: entry
-            retryStrategy:
-              limit: 2
-              retryPolicy: "Always"
-            steps:
-            - - name: prereq-namespace
-                template: submit-ns
-              - name: prereq-serviceaccount
-                template: submit-sa
-              - name: prereq-configmap
-                template: submit-cm
-              - name: prereq-clusterrole
-                template: submit-cr
-              - name: prereq-clusterrolebinding
-                template: submit-crb
-
-          - name: submit-ns
-            resource:
-              action: apply
-              manifest: |
+            script:
+              image: docker.intuit.com/dev/patterns/kubernetes/dev/kubectl-awscli:v1.20.11
```

Comment:
We cannot expose this registry image, this is corp. specific.

Reply:
need remove this, can use public docker.cr registry

Reply:
Fixed https://github.com/keikoproj/addon-manager/pull/99/files#diff-621f750b2c2ce9121b881450f2cd141af138d68e15c29b7b17cfa356474a5375R30